scripts/e2e/external/utils

```diff
@@ -65,7 +65,7 @@ function setup_external() {
 
     # Create external test container on pseudo-ext network
     docker run --cap-add=NET_ADMIN --name "${EXTERNAL_APP}" --network "${EXTERNAL_NET}" \
-        -d registry.access.redhat.com/ubi7/ubi python -m SimpleHTTPServer 80
+        -d registry.access.redhat.com/ubi7/ubi python -m SimpleHTTPServer 1234
```

Comment:
@mkolesnik I had a look at it and did some changes to use `quay.io/submariner/nettest` instead of `registry.access.redhat.com/ubi7/ubi` image in the docker external container (which simulates an external machine) and also changed the ports to 8080 as the simpleserver uses hardcoded 8080 port. However, I'm seeing many errors. 

```
Summarizing 6 Failures:

[Fail] [external-dataplane] Connectivity when connected from a gateway-cluster when a pod connects via TCP to/from a remote service when the pod is on a gateway-node [It] should be able to connect from/to an external app to/from a pod in a cluster 
/go/src/github.com/submariner-io/submariner/test/external/dataplane/connectivity.go:258

[Fail] [external-dataplane] Connectivity when connected from a non-gateway-cluster when a pod connects via TCP to/from a remote service when the pod is not on a gateway [It] should be able to connect from/to an external app to/from a pod in a cluster 
/go/src/github.com/submariner-io/submariner/vendor/github.com/submariner-io/shipyard/test/e2e/framework/docker.go:105

[Fail] [external-dataplane] Connectivity when connected from a non-gateway-cluster when a pod connects via TCP to/from a remote pod when the pod is not on a gateway [It] should be able to connect from/to an external app to/from a pod in a cluster 
/go/src/github.com/submariner-io/submariner/test/external/dataplane/connectivity.go:260

[Fail] [external-dataplane] Connectivity when connected from a non-gateway-cluster when a pod connects via TCP to/from a remote service when the pod is on a gateway [It] should be able to connect from/to an external app to/from a pod in a cluster 
/go/src/github.com/submariner-io/submariner/test/external/dataplane/connectivity.go:260

[Fail] [external-dataplane] Connectivity when connected from a gateway-cluster when a pod connects via TCP to/from a remote pod when the pod is on a gateway-node [It] should be able to connect from/to an external app to/from a pod in a cluster 
/go/src/github.com/submariner-io/submariner/test/external/dataplane/connectivity.go:258

[Fail] [external-dataplane] Connectivity when connected from a non-gateway-cluster when a pod connects via TCP to/from a remote pod when the pod is on a gateway [It] should be able to connect from/to an external app to/from a pod in a cluster 
/go/src/github.com/submariner-io/submariner/test/external/dataplane/connectivity.go:260

Ran 6 of 28 Specs in 286.402 seconds
FAIL! -- 0 Passed | 6 Failed | 7 Pending | 15 Skipped
--- FAIL: TestE2E (286.42s)
```


The docker external container code uses yum and even after resolving it, the tests are failing with some new error code.

```
        should be able to connect from/to an external app to/from a pod in a cluster [It]
        /go/src/github.com/submariner-io/submariner/test/external/dataplane/connectivity.go:90
        Expected
            <string>: 
        to match regular expression
            <string>: 10.131.2.6 .*GET /e2e-tests-ext-dataplane-glr6ccluster2 .*

        /go/src/github.com/submariner-io/submariner/test/external/dataplane/connectivity.go:260
```

Do you mind if I handle the image switching in a separate PR?

On a high level we require the following changes:
1. Modify the https://github.com/submariner-io/submariner/blob/devel/scripts/e2e/external/utils code to use the `quay.io/submariner/nettest` image with simpleserver.
2. Avoid calling yum inside the above code and also tweak it to work with simpleserver.
3. Modify all the images used as part of external-net tests (which basically use `framework.CustomPod`) to use the nettest image instead of `registry.access.redhat.com/ubi7/ubi:latest` image.
4. Identify how to pass the SUBM_IMAGE_REPO and SUBM_IMAGE_TAG to the https://github.com/submariner-io/submariner/blob/devel/scripts/e2e/external/utils script.
5. Maybe more which I'll know when testing.

Reply:
Sounds good, do you mind opening an issue with the details you identified so we don't forget?

Reply:
Done - https://github.com/submariner-io/submariner/issues/2202